examples/baselines/beta_playground/baselines.sh

```diff
@@ -0,0 +1,198 @@
+# Baseline results for beta playground
+
+seeds=(9351 4796 1788)
+file_name='sac_moe'
+
+### State Based Beta Playground Baselines ###
+for seed in ${seeds[@]}
+do
+  python ${file_name}.py --env_id="PushCube-v1" \
+    --num_envs=32 --utd=0.5 --buffer_size=500_000  \
+    --total_timesteps=1_000_000 --eval_freq=50_000 --control-mode="pd_ee_delta_pos" \
+    --exp-name="${file_name}-PushCube-v1-state-${seed}" \
+    --track
+done
+
+# pick cube tests for ensuring manipulation robots work
+for seed in ${seeds[@]}
+do
+  python ${file_name}.py --env_id="PickCube-v1" --seed=${seed} \
+    --num_envs=32 --utd=0.5 --buffer_size=500_000  \
+    --total_timesteps=1_000_000 --eval_freq=50_000 --control-mode="pd_ee_delta_pos" \
+    --exp-name="${file_name}-PickCube-v1-state-${seed}" \
```

Comment:
can you add "-walltime_efficient" or "-sample-efficient" to the end of the experiment names to indicate if these configurations are designed for fast trainign speed (wall time) or good sample efficiency? With 32 environments parallel I am guessing this is not for sample efficiency.